docs/2_1_common_api.adoc

```diff
@@ -960,12 +960,12 @@ In every state, the general form of the FMU equations are:
 
 [latexmath]
 ++++
-\mathbf{v}_{unknown} = \mathbf{h}(\mathbf{v}_{known}, \mathbf{v}_{rest}),
+\mathbf{v}_{\mathit{unknown}} = \mathbf{h}(\mathbf{v}_{\mathit{known}}, \mathbf{v}_{\mathit{rest}}),
 ++++
 
 where
 
-* latexmath:[\mathbf{v}_{unknonwn}] is the vector of unknown floating point variables computed in the actual state:
+* latexmath:[\mathbf{v}_{\mathit{unknonwn}}] is the vector of unknown floating point variables computed in the actual state:
```

Comment:
Typo here: is it on purpose that we're keeping that typo?

Reply:
fixed